source/mainnet/smart-contracts/guides/quick-start.rst

```diff
@@ -80,7 +79,7 @@ Before starting, it is a good idea to read the :ref:`Smart contracts best practi
 
         $cargo concordium build
 
-    This uses Cargo_ for building, but runs further optimizations on the result. Running the ``cargo concordium build`` command will produce a smart contract module which can be found relative to your project root folder in ``./target/concordium/wasm32-unknown-unknown/release/my_module.wasm.v1``. Alternatively, you can supply the location where to store the smart contract module using the ``--out`` option. For example running the following command will output your smart contract module into the root folder of your project in a file name ``my_module.wasm.v1``.
+    This uses Cargo_ for building, but runs further optimizations on the result. Running the ``cargo concordium build`` command will produce a smart contract module which can be found relative to your project root folder in ``~./concordium-out/module.wasm.v1~`` by default. Alternatively, you can supply the location where to store the smart contract module using the ``--out`` option. For example running the following command will output your smart contract module into the root folder of your project in a file name ``my_module.wasm.v1``.
```

Comment:
The folder here is incorrect. I definitely think that cargo-concordium _should_ do that, but it is not the case presently.

Reply:
Added an issue about it: https://github.com/Concordium/concordium-smart-contract-tools/issues/155

Reply:
So what should I do in the meantime? I presume this won't be fixed anytime soon.

Reply:
I would just keep the old folder. Then we can change the docs if/when we change the tooling.